libraries/provision/ansible/playbooks/tasks/stop-sync-gateway.yml

```diff
@@ -1,7 +1,8 @@
 # Stop service
+
 - name: SYNC GATEWAY | Stop sync_gateway service
   service: name=sync_gateway state=stopped
```

Comment:
Same as above comment. This did not work on debian, I believe it is same for ubuntu. Can you double check?